tests/RobotFramework/tests/cumulocity/custom_operation/custom_operation_command/command_handler.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+info() {
+    echo "$(date --iso-8601=seconds 2>/dev/null || date -Iseconds) : INFO : $*" >&2
+}
+
+# Parse the message
+COMMAND="${1}"
+
+# Check if command is wrapped with quotes, if so then remove them
+if [[ "$COMMAND" == \"*\" ]]; then
+    COMMAND="${COMMAND:1:-1}"
+fi
```

Comment:
Since you are running the command using `bash -c "$COMMAND"`, this check is useless.

```suggestion
```

Reply:
Done in 934f052